distribution/feature-packs/server-feature-pack/src/main/resources/licenses/rh-sso/Select2,Apache Software License 2.0.txt

```diff
@@ -0,0 +1 @@
+Select2,GNU General Public License v2.0 only.txt
```

Comment:
I would imagine you pick the more liberal license here (Apache 2) over the more restrictive (GPLv2).  This seems to indicate opposite.